packages/base-driver/lib/protocol/protocol.js

```diff
@@ -237,6 +238,11 @@ function validateExecuteMethodParams(params, paramSpec) {
   } else {
     paramSpec.required ??= [];
     paramSpec.optional ??= [];
+    const unknownNames = _.difference(_.keys(args), paramSpec.required, paramSpec.optional);
+    if (!_.isEmpty(unknownNames)) {
+      log.info(`The following script arguments are not known and will be ignored: ${unknownNames}`);
```

Comment:
would probably make this a warning as it indicates the user is doing something wrong

Reply:
not necessary. We are doing it ourselves in clients to unify the endpoint for various clients